src/Input/TextInputEXT.cs

```diff
@@ -13,6 +13,8 @@
 
 namespace Microsoft.Xna.Framework.Input
 {
+	public delegate void TextEditingEventHandlerEXT (string text, int start, int length);
```

Comment:
Can this be replaced with Action<string, int, int>? We try to expose as few types as possible, so if not this may have to go inside TextInputEXT to keep everything inside the static class.